src/js/middleware.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+// Base grade for express middleware modules...
+//
+// The express module will automatically attempt to wire in anything with this gradeName into itself
+
+var fluid = fluid || require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.express.middleware");
+
+gpii.express.middleware.complainAboutMissingFunction = function(){
```

Comment:
The grunt task does not return any errors.  I verified that `Gruntfile.js` is pointing at the files to be linted.  I also verified the configuration in `.jshintrc` against infusion and other projects.  There are no differences I can see.

I did notice that the grunt jshint plugin is a tilded dependency, which means we may be using different versions of jshint depending on when `npm install` is run.  My copy had 0.9.2.  I removed the tilde and reinstalled to pick up 0.9.0 explicitly.  It didn't help.

I would appreciate help in tracking this down, as I have been rigorously cleaning up stuff like this and don't like invisible errors.